docker-compose.yml

```diff
@@ -32,6 +32,56 @@ services:
   rosie:
     image: serenata/rosie
 
+  prometheus:
+    image: arthursens/serenata-prometheus
+    ports: 
+    - 9090:9090
+    links:
+    - node_exporter:node_exporter
+    - process_exporter:process_exporter
+    - postgres_exporter:postgres_exporter
+    - bi_exporter:bi_exporter
+
+  grafana:
+    image: arthursens/serenata-grafana
+    links:
+    - prometheus:prometheus
+    - postgres:postgres
+    ports:
+    - 3000:3000
+
+  node_exporter:
+    image: prom/node-exporter
+
+  process_exporter:
```

Comment:
This agent is responsible to expose metrics about resources used by every process that runs in your host.
More information about it can be found at [this repository](https://github.com/ncabatoff/process-exporter)